peps/pep-0744.rst

```diff
@@ -525,7 +546,12 @@ likely to be a real concern.
 Not much effort has been put into optimizing the JIT's memory usage yet, so
 these numbers likely represent a maximum that will be reduced over time.
 Improving this is a medium priority, and is being tracked in `GH-116017
-<https://github.com/python/cpython/issues/116017>`__.
+<https://github.com/python/cpython/issues/116017>`__. However, we are currently
+exploring how garbage collection of cold traces (i.e. traces that are no longer
+executed) could be implemented, which could reduce the JIT's memory usage. We may 
+also consider exposing configurable parameters for limiting memory consumption in the
+future. No official APIs will be exposed until the JIT meets the requirements to be
+considering non-experimental.
```

Comment:
Similar here. I think readers probably care more about the "what" than the "how":

```suggestion
<https://github.com/python/cpython/issues/116017>`__.
 
We may consider exposing configurable parameters for limiting memory consumption in the
future, but no official APIs will be exposed until the JIT meets the requirements to be
considering non-experimental.
```